internal/transactions/build.go

```diff
@@ -52,12 +53,15 @@ var BuildCommand = &command.Command{
 
 		globalFlags.Filter = "payload"
```

Comment:
Not sure when this was changed but we at least need to update the docs if this will be the new default value and I believe we should have a value that can be used to display the whole transaction without filters, the reason behind this is that when you build a transaction you might want to confirm all the values and it is hard to see that from the payload

Reply:
Ah was this not intentional? https://github.com/onflow/flow-cli/commit/498221dddc2d77d65dd1ab40cbea0c43a599aac0